uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/task.service.js

```diff
@@ -3,7 +3,8 @@
 define([
   'common/angular',
   'common/lodash',
-  'common/moment'
+  'common/moment',
+  'common/services/crm-ang.service'
```

Comment:
It does not seem like you use this service in that file, any reason why the injection is here?

Reply:
It does actually unless i am adding it in a wrong place. Without it, the `loadForm` is not recognized and my tests fail.

Reply:
This looks wrong to me. I think the service dependency should be here actually:

*uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/tasks-assignments.services.js* cc @AkA84